### ISSUE TITLE
Why should I switch from Atheme?

### ISSUE THREAD

**Issue by patoupatou** [shalture/shalture#72]
I'm just asking, it is said in the README that Atheme is a drying project, care to elaborate on that?

Apart from the fact that this project supplies a Dockerfile (which is why I'm interested in it, as we "dockerized" most of our services on our server), what else does it offer that could convince me to switch? Please note that  I'm not attached to Atheme by any means, even though we used it for years.

Thank you!

**Comment by antoligy** [comment]
Honestly I'm completely of the same mindset - there's a number of these forks, and they each seem to have made changes so I'm left feeling unsure as to which is worth going with.

What does this offer, and, are there plans to begin merging fixes and features from other Atheme forks?

**Comment by kaniini** [comment]
Shalture is at this point mostly an obsolete project as the development has merged back into mainline Atheme.  When Atheme had a maintainership problem (nobody was interested in further development with actual deliverable goals), we initiated a project to encourage forks and evaluate maintenance strategies for a potential future release cycle of Atheme; Shalture was one of the projects that participated in that process.

So really you should probably stick with Atheme, but Shalture may still have a couple of unintegrated changes.

**Comment by GLolol** [comment]
To clarify, the parts about "atheme dying" were there from *before* atheme was rebooted with a new team; the descriptions just haven't been updated yet.

**Comment by patoupatou** [comment]
Thank you guys for the clarifications. I still think the mainline Atheme should merge that Dockerfile though, as most people are starting to use that and I think it's perfect for deploying it quickly.

**Comment by kaniini** [comment]
That would imply that we endorse Docker; we do not.

**Comment by GLolol** [comment]
Stuff like this should go in contrib?

**Comment by kaniini** [comment]
My understanding is that including a Dockerfile outside of the root is pointless because docker's tools wont be able to see/use it.  So not really sure of the point.

**Comment by antoligy** [comment]
The Dockerfile currently is aimed at the root of the project, which in itself is already kinda bad, as really instead it should be pointed to the latest GH tag.

If there's enough interest / it makes sense to go in contrib, I'd be happy to PR to contrib an updated version which would work without being in the main repo.

**Comment by maxteufel** [comment]
If it can be placed in `contrib/`, it says it's not official and it prompts for a custom config in some way (not like <https://github.com/shalture/shalture/blob/master/Dockerfile#L16-L17>), then I'd be ok with it.

**Comment by patoupatou** [comment]
I agree with the not official part, but why the custom config prompt? I am sorry if I am missing the point, but Isn't mounting a conf directory good enough?  This could be documented.

**Comment by maxteufel** [comment]
> but why the custom config prompt?

The default config, which most likely doesn't work in all setups shouldn't be deployed.

> Isn't mounting a conf directory good enough

As that apparently solves above mentioned problem, that would be fine too. Not having read much about docker, but it actually looks like a better idea as changing the config during runtime is possible then.

Not completely sure, but I'd also say if it's solved using a mounted volume, `services.db` and the motd file should be included too.
